docs/api/api.yml

```diff
@@ -300,6 +300,8 @@ paths:
       description: |
         The client is only allowed to update the deposit from DRAFT to SUBMITTED or
         from REJECTED to DRAFT. Other state transitions will be effected by back-end processes.
+        The client must have called `GET /deposit/{id}/doi` and `PUT /deposit/{id}/metadata`
```

Comment:
something like this?

In order to submit, the deposit needs to be valid. In particular, the json metadata MUST be valid, with all mandatory fields specified, among which the Digital Object Identifier (DOI).